server/models/service-instance.js

```diff
@@ -321,6 +321,23 @@ module.exports = function extendServiceInstance(ServiceInstance) {
   }
   ServiceInstance.prototype.logDump = logDump;
 
+  [
+    'agentTraces', 'expressUsageRecords', 'serviceMetrics',
```

Comment:
This would be more maintenance friendly if it were 1 item per line (and with the trailing comma on the last item).